apps/web/public/static/locales/en/common.json

```diff
@@ -2036,6 +2036,8 @@
   "description_requires_booker_email_verification": "To ensure booker's email verification before scheduling events",
   "requires_confirmation_mandatory": "Text messages can only be sent to attendees when event type requires confirmation.",
   "organizations": "Organizations",
+  "upload_cal_video_logo":"Upload Cal Video Logo",
+  "cal_video_logo_upload_instruction":"Please only update a white mono-color version of your logo as an *.svg file on a transparent background",
```

Comment:
This description is not clear to me - instead of `update` we should write `Please only upload`, right?

Also, it says `.svg`, but also `.png` can have a transparent background, and why only `white mono-color`? So I would instead phrase `cal_video_logo_upload_instruction` as:

"It is recommended to upload a logo with a transparent background in either PNG or SVG file format."

What do you think?

Reply:
white mono color because cal video is only in dark mode and logo won't be visible in let's say black color

Reply:
Gotcha, how about then phrasing it as:

To ensure your logo is visible against Cal video's dark background, please upload a light-colored image in PNG or SVG format to maintain transparency.